### PR TITLE
Update device-onboarding-mdm.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note stating that if MDE is configured we can use Turn on Device onboarding and OMA URI is no longer needed.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7132

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/device-onboarding-mdm.md [hunk 1]
```diff
@@ -57,13 +57,17 @@ For security reasons, the package used to Offboard devices will expire 30 days a
...
+
 6. Use the Microsoft Intune custom configuration policy to deploy the following supported OMA-URI settings.
 
     ```text
     OMA-URI: ./Device/Vendor/MSFT/WindowsAdvancedThreatProtection/Offboarding
     Date type: String
     Value: [Copy and paste the value from the content of the DeviceCompliance_valid_until_YYYY-MM-DD.offboarding file]
     ```
+> [!NOTE]
+> if Microsoft Defender for endpoint is already configured, you can "Turn on device onboarding" and Step 6 is no longer required.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/device-onboarding-mdm.md]:
```suggestion
> If Microsoft Defender for Endpoint is already configured, you can **Turn on device onboarding** and Step 6 is no longer required.
```

File: microsoft-365/compliance/device-onboarding-mdm.md [hunk 2]
```diff
@@ -57,13 +57,17 @@ For security reasons, the package used to Offboard devices will expire 30 days a
...
 
 For more information on Microsoft Intune policy settings see, [Windows 10 policy settings in Microsoft Intune](/intune/deploy-use/windows-10-policy-settings-in-microsoft-intune).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/device-onboarding-mdm.md]:
```suggestion
For more information on Microsoft Intune policy settings, see [Windows 10 policy settings in Microsoft Intune](/intune/deploy-use/windows-10-policy-settings-in-microsoft-intune).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**